applications/h264/h264_endoscopy_tool_tracking/CMakeLists.txt

```diff
@@ -67,4 +67,10 @@ if(BUILD_TESTING)
   set_tests_properties(h264_endoscopy_tool_tracking_cpp_test PROPERTIES
                        PASS_REGULAR_EXPRESSION "Deactivating Graph"
                        FAIL_REGULAR_EXPRESSION "[^a-z]Error;ERROR;Failed")
+
+  # For aarch64 LD_LIBRARY_PATH needs to be set
+  if(CMAKE_SYSTEM_PROCESSOR STREQUAL aarch64 OR CMAKE_SYSTEM_PROCESSOR STREQUAL arm64)
+    set_tests_properties(h264_endoscopy_tool_tracking_cpp_test PROPERTIES ENVIRONMENT
+                    "LD_LIBRARY_PATH=/usr/lib/aarch64-linux-gnu/tegra/")
+  endif()
```

Comment:
Is this for container testing only? Why is `LD_LIBRARY_PATH` not set correctly in the container environment?

Is `LD_LIBRARY_PATH` already empty such that no paths are populated by default, or is this replacing other environment paths?

Should we set this in `run` instead, as we do with `PYTHONPATH`?

Reply:
This is just for H264 on arm64 (documented on the readme). I'm not sure about the other path for LD_LIBRARY_PATH.